src/map/script.cpp

```diff
@@ -25736,6 +25736,20 @@ BUILDIN_FUNC( openstylist ){
 #endif
 }
 
+BUILDIN_FUNC(getitempos) {
+	TBL_PC *sd;
+	if (script_rid2sd(sd)){
+		if( current_equip_item_index == -1 ){
```

Comment:
Please add an error message here.
For example:
ShowError( "buildin_getitempos: Invalid usage detected. This command should only be used inside item scripts.\n" );